### PR TITLE
Create 2019-01-23-framework-release-v1363.md

### PR DESCRIPTION
**Pull request by shortjared**

### CODE REVIEW

File: posts/2019-01-23-framework-release-v1363.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - news
+authors: 
+  - JaredShort
+---
+
+The regular cadence of releases is full steam ahead as we release another version of the framework, v1.36.3, brings 4 bug fixes and 9 enhancements.
+
+#### Bug Fixes
+
+- [#5686](https://github.com/serverless/serverless/pull/5686) AWS: Fix stage name validation timing and allow hyphen <a href="https://github.com/serverless/serverless/pull/5686/files?utf8=✓&diff=split" style="text-decoration:none;"> <span style="color:#28a647">+156</span>/<span style="color:#cb2431">-57</span></a> <a href="https://github.com/exoego"> <img src='https://avatars2.githubusercontent.com/u/127635?v=4' alt='' style="max-height:20px"> exoego</a>
```
Comment (dschep) [inline posts/2019-01-23-framework-release-v1363.md]: Hmm. the avatars look huge in the github preview. They aren't in #826. Is this bc of the move to `style` tag? Will it work right on the site even tho github doesn't like it?

Comment (shortjared) [reply posts/2019-01-23-framework-release-v1363.md]: Jeesh. I need to update that it looks like.

Comment (shortjared) [reply posts/2019-01-23-framework-release-v1363.md]: @dschep better now?

Comment (dschep) [reply posts/2019-01-23-framework-release-v1363.md]: Yup!

**Pull request merged**